modules/ossm-jaeger-config-elasticsearch.adoc

```diff
@@ -0,0 +1,77 @@
+////
+This TASK module included in the following assemblies:
+-assembly.adoc
+////
+
+[id="ossm-jaeger-config-elasticsearch_{context}"]
+= Configuring Elasticsearch for Jaeger
+
+The Jaeger collector requires persistent storage. Jaeger can be configured for different storage backends:
+
+* *Memory* - Simple in-memory storage, only recommended for development, demo, or testing purposes.  This is the default option for the `AllInOne` deployment strategy.  Do NOT use for production environments.
+* *Elasticsearch* - For production use.  This is the default option for the `Production` deployment strategy.
+
+NOTE: For this release the only supported storage option for production use is Elasticsearch.
+
+You should modify the default Elasticsearch configuration to match your use case.  You can adjust both the CPU and memory limits for each component by modifying the resources block with valid memory and CPU values.
+
+* For `cpu` the resource quantity is specified in units of cores (e.g., 200m, 0.5, 1).
+* For `memory` it is specified in units of bytes (e.g., 200Ki, 50Mi, 5Gi).
+
+[source,yaml]
+----
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: simple-prod
+spec:
+  strategy: production
+  storage:
+    type: elasticsearch
+    elasticsearch:
+      nodeCount: 1
```

Comment:
Should use the default parameters now used in Maistra:
nodeCount: 3 (https://github.com/Maistra/istio-operator/blob/maistra-1.0/helm/istio/charts/tracing/templates/jaeger-production-elasticsearch.yaml#L66)

16Gi memory, and cpu 1: https://github.com/Maistra/istio-operator/blob/maistra-1.0/helm/istio/charts/tracing/templates/jaeger-production-elasticsearch.yaml#L83-L87